com.stansassets.scene-management/Editor/Visualizer/Interface/IStateStackVisualizerController.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace StansAssets.SceneManagement
+{
+    interface IStateStackVisualizerController<T> : IStateStackVisualizerController, IApplicationStateDelegate<T>
+        where T : Enum
+    {
+    }
+
+    interface IStateStackVisualizerController
+    {
+        string StackName { get; }
+        bool IsBusy { get; }
+        bool IsActive { get; }
```

Comment:
I didn't find any purpose of having these 2 properties.
You can provide only **Stack** for now and we can open some data later.